bundler/doc/contributing/REPRODUCTION_SCRIPT_GUIDE.md

```diff
@@ -0,0 +1,34 @@
+# Reproduction Script Guide
+
+Having a reproduction script of an issue helps us a lot with processing it faster. We know some of our contributors
+could not be familiarized with building reproduction scripts, so in this guide, we'll help you understand what is a 
+reproduction script and provide a template and a couple of examples of some reproduction scripts.
+
+## What is a reproduction script?
+
+A reproduction script is basically a script that reproduces an event. Usually, they are used to reproduce issues, bugs, 
+errors and/or warnings.
+
+The core idea behind creating a reproduction script is to configure the different environment variables to be as similar as

```

Comment:
Perhaps the wording "the different environment variables" could be "the environment".

(Other things than ENV can act on the execution environment of the repro script. We want to avoid confusion, here.)

Reply:
Solved in commit abdeade.

Thanks for the suggestion.